modules/gitops-argo-rolouts-cli-ovr.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+:_mod-docs-content-type: CONCEPT
+[id="gitops-argo-rollouts-cli-overview_{context}"]
+= Argo Rollouts CLI Overview
+
+Using the Argo Rollouts CLI plugin is optional because you can deploy and monitor an application by installing Argo Rollouts using the {gitops-title} Operator. Following the installation of Argo Rollouts, users have the choice to utilize the Argo Rollouts CLI plugin. This plugin effectively handles and displays Argo Rollouts resources directly from the command line interface.
+
+The Argo Rollouts CLI plugin interacts with the Argo Rollouts controller and Argo Rollouts resources. The Argo Rollouts CLI plugin enables you to make changes to an Argo Rollouts image, monitor the progress of an Argo Rollouts promotion, proceed with the promotion steps in a canary deployment, and terminate a failed Argo Rollouts deployment.
```

Comment:
> The Argo Rollouts CLI plugin enables you to make changes to an Argo Rollouts image, monitor the progress of an Argo Rollouts promotion, proceed with the promotion steps in a canary deployment, and terminate a failed Argo Rollouts deployment.

I would add this to the end:

The Argo Rollouts CLI plugin interacts with the Argo Rollouts controller and Argo Rollouts resources. The Argo Rollouts CLI plugin enables you to make changes to an Argo Rollouts image, monitor the progress of an Argo Rollouts promotion, proceed with the promotion steps in a canary deployment, ~and~ terminate a failed Argo Rollouts deployment, _**and more**_.
(addition is _**bolded**_, and likewise see the ~strikethrough~)

(This makes it clear that the CLI has many many commands/use cases, and it is not just limited to those explicitly listed.)

Reply:
Fixed.